cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaRoller.java

```diff
@@ -817,7 +817,7 @@ private void maybeTcpProbe(PodRef podRef, Exception executionException, RestartC
      * @param port The port
      * @throws IOException if anything went wrong.
      */
-    void tcpProbe(String hostname, int port) throws IOException {
+    protected void tcpProbe(String hostname, int port) throws IOException {
```

Comment:
Does it really need to be `protected`? Wouldn't default (with `/* test */` marker) be enough?

Reply:
Yep, not sure why I changed it. Changed to default with the marker.